### PR TITLE
Week 5: Exercise 5.4 JEST test cases

### PR DESCRIPTION
**Pull request by apandyamob**
image

### CODE REVIEW

File: Week-5/Exercise-5.4/calculator.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    expect(result).toBe(8);
+  });
+
+  test("subtracting -2 from 10 should return 12", () => {
+    const result = mathOperations.diff(10, -2);
+
+    expect(result).toBe(12);
+  });
+
+  test("multiplying 2 and 8 should return 16", () => {
```
Comment (RajeshSivanesan) [inline Week-5/Exercise-5.4/calculator.test.js:42]: @AakashPandya Can you please add few more test cases to check for arguments not sent, only one argument is sent etc. Edge case

**Review by RajeshSivanesan — APPROVED** [review]

File: Week-5/Exercise-5.4/calculator.test.js (hunk 1 above)
Comment (apandyamob) [reply Week-5/Exercise-5.4/calculator.test.js:42]: @RajeshSivanesan done
